### PR TITLE
Make meditor category tabs horizontally scrollable, remove unused variable

### LINKED ISSUE #905: Make meditor category tabs horizontally scrollable
Currently, the meditor category headers are cut off on small window sizes
image (2)

### PR DESCRIPTION
**Pull request by mvandenburgh**
Closes #905

### CODE REVIEW

File: web/src/components/Meditor.vue [hunk 1]
```diff
@@ -148,7 +149,6 @@
             class="px-7 py-5 overflow-y-auto"
           >
             <v-jsf
-              ref="vjsfRef"
```
Comment (AlmightyYakob) [inline web/src/components/Meditor.vue:151]: Why was this ref removed? Seems unrelated

Comment (mvandenburgh) [reply web/src/components/Meditor.vue:151]: Sorry, I forgot to mention that, it probably should've been in a separate PR. It's an unused variable.

**Review by AlmightyYakob — APPROVED** [review]
